modules/examples/todolist-service/src/main/scala/todo/runtime/handlers/AppRepositoryHandler.scala

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Related to my previous comments., these handlers can be also placed inside the persistence package.

`todo.persistence.runtime` perhaps? We can omit the `handlers` part.

Reply:
Since it is an optional implementation, I think it can be place also in `persistence` package.

Thank you! :+1: